CHANGELOG.md

```diff
@@ -5,9 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### [Unreleased][unreleased]
 
 #### Fixed
+- [#458]: Java deserialization vulnerability in SerializationSessionDataTranscoder.decode()
```

Comment:
To have a nice link is necesary to declare the link of `[#ABC]:` to the end of file (it's a list with all issues references).